sdk/keyvault/azure-keyvault-certificates/tests/perfstress_tests/get_certificate.py

```diff
@@ -27,12 +26,12 @@ def __init__(self, arguments):
     async def global_setup(self):
         """The global setup is run only once."""
         await super().global_setup()
-        await self.async_client.create_certificate("livekvtestperfcert", CertificatePolicy.get_default())
+        await self.async_client.create_certificate("livekvtestgetcertperfcert", CertificatePolicy.get_default())
```

Comment:
This literal is repeated enough that I'd put it in a variable somewhere. And if you're concerned about conflicts you could do e.g.
```py
def __init__(self, arguments):
    self.cert_name = str(uuid.uuid4())
```